package/scripts/prmon_plot.py

```diff
@@ -18,21 +19,90 @@
     print('Please install these modules first and then retry. ')
     sys.exit(-1)
 
+# Define the labels/units for beautification
+axisunits = {'vmem':'kb', 'pss':'kb', 'rss':'kb',
+             'utime':'sec', 'stime':'sec', 'wtime':'sec',
+             'rchar':'char','wchar':'char',
+             'read_bytes':'b','write_bytes':'b',
+             'rx_packets':'packets', 'tx_packets':'packets',
+             'rx_bytes':'b','tx_bytes':'b'}
+
+axisnames = {'vmem':'Memory', 
+             'pss':'Memory', 
+             'rss':'Memory',
+             'utime':'CPU-time',
+             'stime':'CPU-time',
+             'wtime':'Wall-time',
+             'rchar':'I/O',
+             'wchar':'I/O',
+             'read_bytes':'I/O',
+             'write_bytes':'I/O',
+             'rx_packets':'Network',
+             'tx_packets':'Network',
+             'rx_bytes':'Network',
+             'tx_bytes':'Network'}
+
+legendnames = {'vmem':'Virtual Memory', 
+               'pss':'Proportional Set Size', 
+               'rss':'Resident Set Size',
+               'utime':'User CPU-time',
+               'stime':'System CPU-time',
+               'wtime':'Wall-time',
+               'rchar':'I/O Read',
+               'wchar':'I/O Written',
+               'read_bytes':'I/O Read',
+               'write_bytes':'I/O Written',
+               'rx_packets':'Network Received',
+               'tx_packets':'Network Transmitted',
+               'rx_bytes':'Network Received',
+               'tx_bytes':'Network Transmitted'}
+
+multipliers = {'SEC': 1.,
+               'MIN': 60.,
+               'HR': 60.*60.,
```

Comment:
I would change the name here to `HOUR` - having a plot labled with units of `HR` is a bit unclear to me.